scalanet/ut/src/io/iohk/scalanet/peergroup/DynamicTLSPeerGroupSpec.scala

```diff
@@ -314,6 +312,51 @@ class DynamicTLSPeerGroupSpec extends AsyncFlatSpec with BeforeAndAfterAll {
         }
     }
   }
+  it should "handle messages with prepended messages id-s" in taskTestCase {
```

Comment:
I don't see from the test where these IDs are prepended, can you add some comments? Are you talking about request IDs, or a discriminator for message types (e.g. `Foo`)?